src/index.html

```diff
@@ -11,6 +11,24 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form" data-qa="big">
+      <label>
+        <input
+          class="form_input"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          >
```

Comment:
Set right indentation
```suggestion
        <input
          class="form_input"
          type="text"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
```